src/components/catalog-filter.tsx

```diff
@@ -0,0 +1,139 @@
+import { CamerasType } from '../types/cameras';
+import { useState, useEffect, } from 'react';
+
+type productTabsProps = {
+  setDiplayedCameras: (isActive: []) => void;
+  diplayedCameras: CamerasType;
+}
+
+function CatalogFilter({diplayedCameras, setDiplayedCameras}: productTabsProps): JSX.Element {
```

Comment:
```suggestion
function CatalogFilter({diplayedCameras, setDiplayedCameras}: productTabsProps) {
```
Нет смысла указывть тип здесь и везде